acs-integration-tests/pom.xml

```diff
@@ -382,7 +385,11 @@
                             <memory>1024</memory>
                             <buildpack>java-buildpack</buildpack>
                             <env>
-                                <SPRING_PROFILES_ACTIVE>integration,predix,cloud-redis,cloudDbConfig</SPRING_PROFILES_ACTIVE>
+                                <SPRING_PROFILES_ACTIVE>integration,predix,cloud-redis,cloudDbConfig,predixAudit</SPRING_PROFILES_ACTIVE>
```

Comment:
We don't seem to be using the `predixAudit` Spring profile anywhere in integration tests so remove it so there's one less reference to remove in a later refactoring?